src/pipelines/epidemiology/pe_authority.py

```diff
@@ -17,21 +17,32 @@
 from lib.cast import safe_int_cast, safe_str_cast
 from lib.data_source import DataSource
 from lib.case_line import convert_cases_to_time_series
-from lib.constants import SRC
-from lib.io import read_file, fuzzy_text
 from lib.time import datetime_isoformat
 from lib.utils import table_multimerge, table_rename
 
 _column_adapter = {
     "FECHA_RESULTADO": "date",
     "FECHA_FALLECIMIENTO": "date",
     "DISTRITO": "subregion2_name",
+    "DEPARTAMENTO": "subregion1_name",
     "PROVINCIA": "province_name",
     "EDAD": "age",
     "EDAD_DECLARADA": "age",
     "SEXO": "sex",
 }
 
+# Used only for departments with mismatching names or special characters
```

Comment:
I thought we had a method that replaced special chars for matching purposes?

Reply:
We do. The problem here are lacking capabilities in our matching method, since it is designed to only match a single field. In this case, both `subregion1_name` and `subregion2_name` need fuzzy matching, but we only have one `string_match` field. Since there is a sufficiently small number of `subregion1_name`'s in PE which require some wrangling, I'm doing it by hand.

This is not an ideal design, but considering that we have been able to make it work so far I think it's a fair compromise.